examples/plugin/wasmedge-zlib/README.md

```diff
@@ -0,0 +1,80 @@
+# WasmEdge WASMEDGE-Zlib example.
+
+This is a example to demonstrate how to use wasmedge-zlib plugin of WasmEdge with C++.
```

Comment:
Please use tools to fix several grammar issues. E.g. grammarly.

Reply:
Done.